x/epochs/keeper/grpc_query_test.go

```diff
@@ -0,0 +1,34 @@
+package keeper_test
+
+import (
+	gocontext "context"
+	"time"
+
+	"github.com/c-osmosis/osmosis/x/epochs/types"
+)
+
+func (suite *KeeperTestSuite) TestQueryEpochInfos() {
+	suite.SetupTest()
+	queryClient := suite.queryClient
+
+	// Invalid param
+	epochInfosResponse, err := queryClient.EpochInfos(gocontext.Background(), &types.QueryEpochsInfoRequest{})
+	suite.Require().NoError(err)
+	suite.Require().Len(epochInfosResponse.Epochs, 2)
+
+	// check if EpochInfos are correct
+	suite.Require().Equal(epochInfosResponse.Epochs[0].Identifier, "daily")
+	suite.Require().Equal(epochInfosResponse.Epochs[0].StartTime, time.Time{})
+	suite.Require().Equal(epochInfosResponse.Epochs[0].Duration, time.Hour*24)
+	suite.Require().Equal(epochInfosResponse.Epochs[0].CurrentEpoch, int64(0))
+	suite.Require().Equal(epochInfosResponse.Epochs[0].CurrentEpochStartTime, time.Time{})
+	suite.Require().Equal(epochInfosResponse.Epochs[0].EpochCountingStarted, false)
+	suite.Require().Equal(epochInfosResponse.Epochs[0].CurrentEpochEnded, true)
+	suite.Require().Equal(epochInfosResponse.Epochs[1].Identifier, "weekly")
+	suite.Require().Equal(epochInfosResponse.Epochs[1].StartTime, time.Time{})
+	suite.Require().Equal(epochInfosResponse.Epochs[1].Duration, time.Hour*24*7)
+	suite.Require().Equal(epochInfosResponse.Epochs[1].CurrentEpoch, int64(0))
+	suite.Require().Equal(epochInfosResponse.Epochs[1].CurrentEpochStartTime, time.Time{})
```

Comment:
Why is start time blank here, shouldn't it be the epoch's start time?

Reply:
same as above. Within test setup, chain's startTime is time.Time{}

Reply:
added